### DISCUSSION TITLE
scroll to top icon

### DISCUSSION THREAD

**Discussion by AntoineSoetewey** [jpanther/congo#147] [Q&A] [answered]
Hello,

When does the scroll to top icon appears?

For example, it appears for this page: https://elsaetantoine.com/hebergement/, but not for this one: https://elsaetantoine.com/histoire/

Thanks in advance!

**Comment by jpanther** [comment, marked as answer]
Hey @AntoineSoetewey, the scroll to top only appears on pages with more than 200 words which is perhaps why it doesn't show on that second page.

**Reply by AntoineSoetewey** [reply]
Thanks!
